### PR TITLE
fixed ssh name for the bastion host

### PR DESCRIPTION
**Pull request by bkarypid**
replaced user name from core to ubuntu in order to successfully ssh to the bastion host in aws private

### CODE REVIEW

File: bootstrap/aws/private-cloud/util.sh [hunk 1]
```diff
@@ -36,11 +36,11 @@ ansible_ssh_config() {
     StrictHostKeyChecking  no
     ServerAliveInterval    120
     TCPKeepAlive           yes
-    ProxyCommand           ssh -q -A -F $(pwd)/ssh.config core@$APOLLO_bastion_ip nc %h %p
+    ProxyCommand           ssh -q -A -F $(pwd)/ssh.config ubuntu@$APOLLO_bastion_ip nc %h %p
```
Comment (enxebre) [inline bootstrap/aws/private-cloud/util.sh]: shouldn't this still be core? Ideally we want to move this box into coreos

Comment (tayzlor) [inline bootstrap/aws/private-cloud/util.sh]: There is a ticket for that here https://github.com/Capgemini/Apollo/issues/604

File: bootstrap/aws/private-cloud/util.sh [hunk 2]
```diff
@@ -36,11 +36,11 @@ ansible_ssh_config() {
...
     ControlMaster          auto
     ControlPath            ~/.ssh/mux-%r@%h:%p
     ControlPersist         30m
-    User                   core
+    User                   ubuntu
```
Comment (tayzlor) [inline bootstrap/aws/private-cloud/util.sh]: i'm not sure both of these need to be ubuntu - one of them needs to be core

**Comment by wallies** [comment]
closing this in favour of the work being done here https://github.com/Capgemini/Apollo/pull/657

**Pull request closed without merging**